src/login/logind-seat.c

```diff
@@ -207,19 +209,107 @@ int seat_preallocate_vts(Seat *s) {
         return r;
 }
 
-int seat_apply_acls(Seat *s, Session *old_active) {
+static void seat_triggered_uevents_done(Seat *s) {
+        assert(s);
+
+        if (!set_isempty(s->uevents))
+                return;
+
+        Session *session = s->active;
+
+        if (session) {
+                session_save(session);
+                user_save(session->user);
+        }
+
+        if (session && session->started) {
+                session_send_changed(session, "Active");
+                session_device_resume_all(session);
+        }
+
+        if (!session || session->started)
+                seat_send_changed(s, "ActiveSession");
+}
+
+int manager_process_device_triggered_by_seat(Manager *m, sd_device *dev) {
+        assert(m);
+        assert(dev);
+
+        sd_id128_t uuid;
+        if (sd_device_get_trigger_uuid(dev, &uuid) < 0)
```

Comment:
maybe debug log?

Reply:
now error code is propagated to the caller and logged there.